playwright-e2e/pages/HomePage.ts

```diff
@@ -56,4 +54,17 @@ export default class HomePage extends BasePage {
     return this.bookingRefNUmber.innerText()
   }
 
-}
+  async clickOnAddPrisoner(): Promise<void> {
+    await this.addPrisoner.click()
+  }
+
+  async selectPrisonFromList(prisonName: string): Promise<void> {
+    const prisonRadio = this.page.locator(`input[type="radio"][name="prisonId"][value="${prisonName}"]`)
+    await prisonRadio.waitFor({ state: 'visible' }) // ensure it's in view
```

Comment:
Surprised you have to wait for this; I'd have thought it would be ready on the page 🤷‍♂️